Content.Server/GameObjects/Components/Fluids/SpillableComponent.cs

```diff
@@ -51,11 +52,17 @@ protected override void Activate(IEntity user, SpillableComponent component)
                     }
 
                     // Need this as when we split the component's owner may be deleted
-                    var entityLocation = component.Owner.Transform.Coordinates;
-                    var solution = solutionComponent.Drain(solutionComponent.DrainAvailable);
-                    solution.SpillAt(entityLocation, "PuddleSmear");
+                    solutionComponent.Drain(solutionComponent.DrainAvailable).SpillAt(component.Owner.Transform.Coordinates, "PuddleSmear");
                 }
             }
         }
+
+        void IDropped.Dropped(DroppedEventArgs eventArgs)
+        {
+            if (Owner.TryGetComponent(out ISolutionInteractionsComponent solutionComponent))
```

Comment:
this should probably only happen if `!eventArgs.intentional`

"dropping" items onto tables on purpose is fine I feel

Reply:
oh wait, yeah, I eh